CONTRIBUTING.md

```diff
@@ -1,36 +1,50 @@
+# Contributing
+
 ## Prerequisites
 
 1. Go: [https://golang.org/dl/](https://golang.org/dl/)
```

Comment:
Imho we don't need to document that Go is needed, then it's also valid to document that you need git, Make and an editor